api/src/test/java/org/pmiops/workbench/db/dao/UserServiceTest.java

```diff
@@ -441,4 +441,23 @@ public void test_hasAuthority_DEVELOPER() {
       assertThat(userService.hasAuthority(user.getUserId(), auth)).isTrue();
     }
   }
+
+  @Test
+  public void test_confirmProfile() {
+    assertThat(providedDbUser.getProfileLastConfirmedTime()).isNull();
+
+    // user confirms profile, so confirmation time is set to START_INSTANT
+
+    DbUser retrievedUser = userService.confirmProfile();
+    assertThat(retrievedUser.getProfileLastConfirmedTime())
+        .isEqualTo(Timestamp.from(START_INSTANT));
+
+    // time passes, user confirms again, confirmation time is updated
+
+    tick();
```

Comment:
was not aware of this function - how far forward does it tick?

Reply:
I didn't know about it either - it's 1000 ms / 1s